ui/src/app/editor/components/pipeline-element-options/pipeline-element-options.component.ts

```diff
@@ -227,4 +226,25 @@ export class PipelineElementOptionsComponent implements OnInit, OnDestroy {
     ngOnDestroy(): void {
         this.pipelineElementConfiguredObservable.unsubscribe();
     }
+
+    deepCopy(obj) {
+        const clone = {};
+
+        if (
+            typeof obj !== 'object' ||
+            typeof obj === undefined ||
+            obj === null ||
+            Array.isArray(obj) ||
```

Comment:
Just a question...why are arrays excluded here?
E.g., when called with a `PipelineElementConfig[]` object (line 164), it seems the method returns the original input and not a deep copy of the list including its elements?